app/views/default/custom.html.erb

```diff
@@ -0,0 +1,9 @@
+<ul>
+<% anonymous_top_menu_items.each_with_index do |item, index| %>
```

Comment:
Outputting the top menu items for the crawler view since we don't quite know what the theme is going to render. At least the top menu gives the crawler a place to start crawling.